keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/Schema.kt

```diff
@@ -0,0 +1,65 @@
+package com.netflix.spinnaker.keel.sql
+
+import org.jooq.Field
+import org.jooq.Record
+import org.jooq.Table
+import org.jooq.impl.DSL
+import org.jooq.impl.DSL.table
+import java.sql.Timestamp
+
+internal sealed class Schema {
```

Comment:
Any thoughts on this way of modelling the schema? I feel kind of like I'm reinventing bits of JOOQ's generated DTO classes (which we're not using currently). Since with this changeset I started referencing tables across the various repositories I thought it made sense to have shared definitions. I also needed qualified field names for join-based selects.